src/Dapr.AspNetCore/BulkSubscribeAttribute.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System;
+
+namespace Dapr.AspNetCore
+{
+    /// <summary>
+    /// BulkSubscribeAttribute describes options for a bulk subscriber with respect to a topic.
+    /// </summary>
```

Comment:
Does this attribute need to be paired with a `[Topic]`? Or can it exist by itself?

Reply:
It needs to be paired with `[Topic]`. Updated the docs.